tests/e2e/historyRouter.spec.ts

```diff
@@ -45,7 +45,7 @@ test.describe("project page navigation", () => {
     const projectHistoryURL = `${sourceBrowsingUrl}/history/${aliceMainHead}`;
     await page.goto(projectHistoryURL);
 
-    await page.locator("text=Add README.md").click();
+    await page.locator("text=Moving and copying some files").click();
```

Comment:
It would be nice to keep the fixture as it is and add this new case in a branch. As the fixture has some internal logic to it, and it might also avoid the need to change all the tests.

I'd suggest we have a new feature branch called `feature/move-copy-files`. And then have the following commits:
  1. create a folder called `copies` with a single file in it `copies/aaa.txt` with the content `AAA` and another folder `moves` with a file `moves/111.txt`  with the content `111`.
  2. create a copy of `copies/aaa.txt` called `copies/bbb.txt` and move `moves/111.txt` to `moves/222.txt`.

Reply:
yeah good idea, did what you said with some minor changes, but reduced the changes in the test suite to a minimum